database/migrations/2024_10_23_104818_implement_database_dl_authorities_on_models.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    public function up(): void
+    {
+        Schema::table('clx_messages', function (Blueprint $table) {
+            $table->renameColumn('datalink_authority', 'datalink_authority_id');
+        });
+
+        Schema::table('cpdlc_messages', function (Blueprint $table) {
+            $table->renameColumn('datalink_authority', 'datalink_authority_id');
+        });
+    }
+
+    public function down(): void
+    {
+        Schema::table('clx_messages', function (Blueprint $table) {
```

Comment:
unnecessary table update, pull what's in here out

Reply:
https://github.com/vatsimnetwork/nattrak/pull/160/commits/a662997a44373e51b4d8d90d1f67da610efcdcd2